src/include/network/routines.rb

```diff
@@ -947,6 +947,18 @@ def SetAllLinksUp
       ret
     end
 
+    def SetAllLinksDown
+      interfaces = GetAllInterfaces()
+      ret = !interfaces.empty?
```

Comment:
copy-paste-edit of similar function for ycp times. I'll give it some touches ...